velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -57,6 +57,85 @@ class DateTimeFunctionsTest : public SparkFunctionBaseTest {
   }
 };
 
+TEST_F(DateTimeFunctionsTest, toUTCTimestamp) {
+  const auto toUTCTimestamp = [&](const std::string_view ts,
+                                  const std::optional<std::string> tz) {
+    auto timestamp = std::make_optional<Timestamp>(
+        util::fromTimestampString(ts.data(), ts.length()));
+    return evaluateOnce<Timestamp>("to_utc_timestamp(c0, c1)", timestamp, tz);
+  };
+  EXPECT_EQ(
+      util::fromTimestampString("2015-07-24 07:00:00"),
+      toUTCTimestamp("2015-07-24 00:00:00", "America/Los_Angeles"));
+  EXPECT_EQ(
+      util::fromTimestampString("2015-01-24 08:00:00"),
+      toUTCTimestamp("2015-01-24 00:00:00", "America/Los_Angeles"));
+  EXPECT_EQ(
+      util::fromTimestampString("2015-01-24 00:00:00"),
+      toUTCTimestamp("2015-01-24 00:00:00", "UTC"));
+  EXPECT_EQ(
+      util::fromTimestampString("2015-01-24 00:00:00"),
+      toUTCTimestamp("2015-01-24 05:30:00", "Asia/Kolkata"));
```

Comment:
nit: what is the error message here, and shall we assert it?

Reply:
I've removed this since the next case `VELOX_ASSERT_THROW` will assert the error message.